res/css/views/elements/_ResizeHandle.scss

```diff
@@ -20,15 +20,6 @@ limitations under the License.
     z-index: 100;
 }
 
-// We'd like to remove this, but this makes matrixchat's resizehandle's
-// negative margin & greater than its positive padding. If it's the same
-// or less, Safari gets confused about overflows somehow and
-// https://github.com/vector-im/element-web/issues/19863 happens.
-.mx_MatrixChat > .mx_ResizeHandle.mx_ResizeHandle_horizontal {
-    margin: 0 -10px 0 0;
```

Comment:
this is moved to the _MatrixChat class since there we also handle the hover behavior. This way all the special resizer logic is in the according class.